completions/bash/sway

```diff
@@ -1,6 +1,6 @@
-# sway(1) completion
+# swayfx(1) completion
 
-_sway()
+_swayfx()
```

Comment:
Shouldn't these file names be renamed to swayfx? (all completion files)

Reply:
I didn't rename any files to avoid creating unnecessary conflicts when pulling from upstream.
The files get renamed by meson at install time (see the changes in `meson.build`).
e.g `completions/bash/sway` gets deployed to `/usr/share/bash-completion/completions/swayfx`